src/diet.ts

```diff
@@ -464,6 +465,32 @@ export function potionMenu(
   const speakeasy = $item`Clan speakeasy`;
   const hasSpeakeasy = getClanLounge()[`${speakeasy}`];
 
+  const twiceHauntedPrice =
+    Math.min(
+      garboValue($item`haunted orange`),
```

Comment:
I should have caught this elsewhere--we don't want to use `getSaleValue` and, by extension, `garboValue` for things we buy. `getSaleValue` multiplies the mallPrice by 0.9 when relevant, which is great for its stated purpose--the mallPrice is the price items _aren't_ selling at--but when we're comparing this against other things to buy, and when we're doing math to determine what things to eat, we decidedly don't want to be doing that.

Reply:
In this particular instance I actually want to value items below their mall value if they see bottle of vodka or orange, but for the others I want mall value. I can write a version of this logic specifically for diet.

I don't think there is much of anything that uses garboValue to purchase except these recent changes